### ISSUE TITLE
Anchors can be added, but not removed?

### ISSUE THREAD

**Issue by machenmusik** [mozilla-mobile/webxr-ios#70]
from Slack:

mchen [12:33 PM]
not sure I am reading this right -- is there really an addAnchor but no removeAnchor(s) ?  https://github.com/mozilla-mobile/webxr-ios/blob/master/XRViewer/WebController/WebController.m

blairmacintyre [1:49 PM]
It might not have been implemented; if there’s nothing in the arkitwrapper for a message like that, please file an issue

**Comment by blairmacintyre** [comment]
The app actually has a call to delete anchors.  There are/were a few problems, though.

1) the app and javascript had some bugs preventing anchors that were deleted in the app from reflecting that up in the framework.  We've fixed that part (will come out in an update soon).  This applied to when ARKit removed anchors (e.g., when planes are no longer tracked)

2) the javascript framework, as you say, doesn't have the appropriate methods.  We've added some of this in the fix above (1) but not all.  Methods will need to be added to the polyfill and to three.xr.js to deal with this.
